src/ASM/ASMs2D.C

```diff
@@ -1274,13 +1274,13 @@ double ASMs2D::getParametricLength (int iel, int dir) const
 #endif
 
   switch (dir)
-    {
-    case 1: return surf->knotSpan(0,nodeInd[inod1].I);
-    case 2: return surf->knotSpan(1,nodeInd[inod1].J);
-    }
+  {
+  case 1: return surf->knotSpan(0,nodeInd[inod1].I);
+  case 2: return surf->knotSpan(1,nodeInd[inod1].J);
+  }
```

Comment:
uhm, all i did was nuke the tabs, but i guess i forgot to set the project tabstop to 8 instead of the default 4 so the autoindenter got confused. fixed.